<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCRC</class>
 <widget class="QDialog" name="DialogCRC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calc CRC</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="line_in"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="line_out"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="bt_copy">
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CRC</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="line_CRC"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Algorithm</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbt_crc16">
        <property name="text">
         <string>CRC16</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbt_crcMODBUS">
        <property name="text">
         <string>CRC(MODBUS RTU)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbt_crcNMEA">
        <property name="text">
         <string>OR(NMEA)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="btBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="bt_compute">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbt_Dec">
        <property name="text">
         <string>Decimal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbt_ASCII">
        <property name="text">
         <string>ASCII</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbt_NMEA">
        <property name="text">
         <string>NMEA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbt_BIN">
        <property name="text">
         <string>BIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbt_HEX">
        <property name="text">
         <string>HEX</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCRC</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCRC</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
